vizro-core/src/vizro/static/css/figures.css

```diff
@@ -1,50 +1,94 @@
 .card-kpi {
-  border-left: 4px solid var(--text-secondary);
+  border-left: 4px solid var(--bs-secondary);
+  height: 100%;
+  padding: 1rem;
+}
+
+/* We currently ship this CSS file with the package, so this needs to live here at the moment. */
+@font-face {
+  font-family: "Material Symbols Outlined";
+  font-style: normal;
+  font-weight: 400;
+  src: url("https://fonts.gstatic.com/s/materialsymbolsoutlined/v199/kJF1BvYX7BgnkSrUwT8OhrdQw4oELdPIeeII9v6oDMzByHX9rA6RzaxHMPdY43zj-jCxv3fzvRNU22ZXGJpEpjC_1v-p_4MrImHCIJIZrDCvHOejbd5zrDAt.woff2")
```

Comment:
This URL looks kind of weird... Where did you get it from?

Reply:
You sent me the link to this actually 😄 

https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined

Reply:
Ah ok. Please can you download the woff2 file to our fonts and then make this relative, like ` url("./fonts/something-that's-not-too-gross")`. Together with a comment on where the woff2 file comes from.

Then I will just check something about how the CDN works and either that's solved or we keep as you have it here.

Reply:
I've tried to do that here https://github.com/mckinsey/vizro/pull/578/commits/e839bc4ce7b4427e0b1a27ce4130e76674e315fe, but it doesn't seem to read the local font file 🤔 

We probably also need to ship the fonts with the package right? Is that what you mean with you want to check how the CDN works?